src/Logging/Logging.Testing/src/Xunit/LoggedTestRunner.cs

```diff
@@ -54,6 +54,12 @@ private async Task<decimal> InvokeTestMethodAsync(ExceptionAggregator aggregator
 
             if (!typeof(LoggedTestBase).IsAssignableFrom(TestClass) || retryAttribute == null)
             {
+                var repeatAttribute = GetRepeatAttribute(TestMethod);
```

Comment:
Hmm should we explicitly prevent the case when Repeat and Retry are both added? Otherwise the failures might be hidden behind retry.